src/tracing/CMakeLists.txt

```diff
@@ -43,9 +43,9 @@ endfunction()
 
 set(osd_traces oprequest.tp osd.tp pg.tp)
 add_tracing_library(osd_tp "${osd_traces}" 1.0.0)
-add_tracing_library(rados_tp librados.tp 2.0.0)
+add_tracing_library(librados_tp librados.tp 2.0.0)
```

Comment:
So at that moment, only target "librados-tp" is built to meet the dependency. But,
unfortunately, there's a another build error following, so "librados_tp" target has not been started to build, right?  
```
Scanning dependencies of target build_rocksdb
[ 17%] rocksdb building
  GEN      util/build_version.cc
  GEN      util/build_version.cc
make[3]: *** No rule to make target 'db/writebuffer.h', needed by 'db/column_family.o'.  Stop.
src/CMakeFiles/build_rocksdb.dir/build.make:57: recipe for target 'src/CMakeFiles/build_rocksdb' failed
make[2]: *** [src/CMakeFiles/build_rocksdb] Error 2
CMakeFiles/Makefile2:424: recipe for target 'src/CMakeFiles/build_rocksdb.dir/all' failed
make[1]: *** [src/CMakeFiles/build_rocksdb.dir/all] Error 2
```